.github/workflows/rover.yaml

```diff
@@ -35,6 +35,8 @@ jobs:
           submodules: true
       - name: Check License
         uses: apache/skywalking-eyes@9bd5feb86b5817aa6072b008f9866a2c3bbc8587
+      - name: Format
+        run: if [ "$(gofmt -s -l . | wc -l)" -gt 0 ]; then exit 1; fi
```

Comment:
What is the difference between this and `make lint` (`make container-lint`)?

Reply:
Ohhh, I have rechecked the lint config(`.golangci.yml`), It already contains the `gofmt`. So It could be checked the format when running the `make lint`. So, close this PR.